osu.Server.Queues.ScoreStatisticsProcessor/Models/UserStatsTaiko.cs

```diff
@@ -3,7 +3,5 @@
 namespace osu.Server.Queues.ScoreStatisticsProcessor.Models
 {
     [Table("osu_user_stats_taiko")]
-    public class UserStatsTaiko : UserStats
-    {
-    }
+    public class UserStatsTaiko : UserStats;
```

Comment:
i dunno about this 😅 

I had

```diff
diff --git a/osu.Server.Queues.ScoreStatisticsProcessor.sln.DotSettings b/osu.Server.Queues.ScoreStatisticsProcessor.sln.DotSettings
index ae6804e..62f57ad 100644
--- a/osu.Server.Queues.ScoreStatisticsProcessor.sln.DotSettings
+++ b/osu.Server.Queues.ScoreStatisticsProcessor.sln.DotSettings
@@ -142,6 +142,7 @@
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=RedundantExplicitParamsArrayCreation/@EntryIndexedValue">WARNING</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=RedundantImmediateDelegateInvocation/@EntryIndexedValue">WARNING</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=RedundantLambdaSignatureParentheses/@EntryIndexedValue">WARNING</s:String>
+	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=RedundantTypeDeclarationBody/@EntryIndexedValue">HINT</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=RedundantTypeSpecificationInDefaultExpression/@EntryIndexedValue">WARNING</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=RedundantLinebreak/@EntryIndexedValue">WARNING</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=RedundantSpace/@EntryIndexedValue">WARNING</s:String>

```

Reply:
/shrug. Feel free to force push that instead, I couldn't be bothered to find the correct resharper incantation for this (and rider severity change thingy didn't want to make it for me).

Reply:
nah it's fine at this point.